messaging/src/main/java/io/lantern/messaging/Messaging.kt

```diff
@@ -850,6 +876,67 @@ class Messaging(
         }
     }
 
+    @Throws(IllegalArgumentException::class)
+    fun introduce(recipientIds: List<String>) {
+        if (recipientIds.size < 2) {
+            throw IllegalArgumentException("Please specify at least 2 recipients to introduce")
+        }
+
+        val introducedParties = recipientIds.map { recipientId ->
```

Comment:
We had agreed we can add ourselves as a contact (in the same way that in Slack you can DM yourself). Can we also send an invite to connect ourselves to another person? Seems useless. What do you think?

Reply:
I think that will be a little redundant, at the end the user `myself` is only our ID but added as a fake contact, and when we send a message we don't use this: https://github.com/getlantern/messaging-android/blob/6c54447c7ab46fa1578f3ff960deba2fca23125a/messaging/src/main/java/io/lantern/messaging/Messaging.kt#L346

Reply:
Great question! I'll wait until @Crdzbird's work is ready and we can evaluate how that impacts this.